src/Logging/Dispatcher.h

```diff
@@ -19,20 +21,30 @@ class Dispatcher {
          */
         ~Dispatcher();
 
+        void begin();
+
+        void enableLogging();
```

Comment:
would it make more sense to have a single method setLoggingEnable(bool value)?

Reply:
Would you prefer that? I'd be happy to make the change, or you can also commit and push changes to this branch yourself and they'll automatically show up in the PR.